Userland/md.cpp

```diff
@@ -32,22 +32,34 @@
 #include <LibMarkdown/Document.h>
 #include <stdio.h>
 #include <string.h>
+#include <sys/ioctl.h>
+#include <unistd.h>
 
 int main(int argc, char* argv[])
 {
-    if (pledge("stdio rpath", nullptr) < 0) {
+    if (pledge("stdio rpath tty", nullptr) < 0) {
         perror("pledge");
         return 1;
     }
 
     const char* file_name = nullptr;
     bool html = false;
+    int view_width = 0;
 
     Core::ArgsParser args_parser;
     args_parser.add_option(html, "Render to HTML rather than for the terminal", "html", 'H');
+    args_parser.add_option(view_width, "Viewport width for the terminal (defaults to current terminal width)", "view-width", 0, "width");
     args_parser.add_positional_argument(file_name, "Path to Markdown file", "path", Core::ArgsParser::Required::No);
     args_parser.parse(argc, argv);
 
+    if (!html && view_width == 0 && isatty(STDOUT_FILENO)) {
```

Comment:
So if `!html && view_width == 0`, but `!isatty(STDOUT_FILENO)`, you're going to continue with 0 (and not say 80)?